app/statistics/server/lib/statistics.js

```diff
@@ -59,8 +59,8 @@ export const statistics = {
 		}
 
 		// User statistics
-		statistics.totalUsers = Meteor.users.find().count();
-		statistics.activeUsers = Meteor.users.find({ active: true }).count();
+		statistics.totalUsers = Users.find().count();
+		statistics.activeUsers = Users.getActiveLocalUserCount();
```

Comment:
This is so it excludes remote and app users right?

Can we add app users to this?  Would be probably useful to know how many app users specifically like we have with remote users

Reply:
Yes, I think we should add the count of app users